README.md

```diff
@@ -32,7 +32,7 @@ Using the provider
 
 Full docs: TODO
 
-Example: Create a repository
+Example: Create a repository with a non-default entitlement
```

Comment:
what's the definition of a `non-default` entitlement and will a user of the provider know what the default is?; an entitlement token that is user-generated instead of those auto-generated by Cloudsmith at repository creation time?

Reply:
Yeah, just one that's explicitly created by the user. In this case it's mostly just a throwaway comment for the user.

The plan is to add proper user-facing documentation before any initial release.